demo/components/presentational/EstateForSale.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Image, StyleSheet, Text } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveHeight,
+} from "react-native-responsive-dimensions";
+
+export default function EstateForSale({ estate }) {
+  return (
+    <React.Fragment>
+      <Image style={styles.landImage} source={{ uri: estate.img }} />
```

Comment:
Maybe split out the with `const { img}  = estate` first?

Reply:
Done.

Reply:
Thanks! Resolving.